test/Swashbuckle.AspNetCore.Newtonsoft.Test/SchemaGenerator/NewtonsoftSchemaGeneratorTests.cs

```diff
@@ -322,10 +322,15 @@ public void GenerateSchema_SetsValidationProperties_IfComplexTypeHasValidationAt
             Assert.Equal(3, schema.Properties["StringWithLength"].MaxLength);
             Assert.Equal(1, schema.Properties["ArrayWithLength"].MinItems);
             Assert.Equal(3, schema.Properties["ArrayWithLength"].MaxItems);
-            Assert.Equal(true, schema.Properties["IntWithRange"].ExclusiveMinimum);
-            Assert.Equal(true, schema.Properties["IntWithRange"].ExclusiveMaximum);
             Assert.Equal("byte", schema.Properties["StringWithBase64"].Format);
             Assert.Equal("string", schema.Properties["StringWithBase64"].Type);
+            Assert.Null(schema.Properties["IntWithRange"].ExclusiveMinimum);
+            Assert.Null(schema.Properties["IntWithRange"].ExclusiveMaximum);
+            Assert.Equal(true, schema.Properties["IntWithExclusiveRange"].ExclusiveMinimum);
+            Assert.Equal(true, schema.Properties["IntWithExclusiveRange"].ExclusiveMaximum);
+#else
+            Assert.Null(schema.Properties["IntWithRange"].ExclusiveMinimum);
```

Comment:
Are these two assertions in both branches? If so, just have them after the `#endif`